Assets/Scripts/SS3D/Systems/Screens/GameScreensController.cs

```diff
@@ -21,6 +21,14 @@ protected override void OnStart()
             _menuOpen = true;
             _blockNone = true;
             _spawnedState = PlayerSpawnedState.IsNotSpawned;
+            SystemLocator.Get<InputSystem>().Inputs.Other.Menu.performed += _ =>
```

Comment:
same as previous comment, I'd raver have a good old method here, but maybe that's just me. What's the reason here for the lambda expression ? Is there any other reason than having the method directly next to the suscribing ?